lib/tasks/clear_orphans.rake

```diff
@@ -5,7 +5,7 @@ namespace :db do
     OrphanList.destroy_all
     PendingOrphan.destroy_all
     PendingOrphanList.destroy_all
-    Sponsorship.destroy_all
+    Sponsorship.delete_all
```

Comment:
I'm normally against comments but perhaps in this case a comment making intent clear might be helpful.